### ISSUE TITLE
Save port number for Cookies / Auth problems on multiple instances

### ISSUE THREAD

**Issue by praul** [jonaswinkler/paperless-ng#122]
tldr: It would be great if paperless-ng were to set cookies for host:**port**. I have two instances running on separate ports and therefore have to reauth constantly .

______

Hey, first of all, thank you very much. I really like paperless-ng and use it quite a lot.
As of yesterday, I have two instances of paperless running to separate my documents from those of my wife (It gets confusing with archive numbers if everything is on one instance.)

But there is a problem with cookie / auth management when working simultaneously on both instances. I constantly get logged out on one or the other instance. paperless-ng sets cookies for domain, discarding the port number.

A fix would be highly appreciated. Thank you

**Comment by jonaswinkler** [comment]
I've looked into that and it seems I can tell the framework used for the backend what name to use for cookies. This is a manual process and you'd have to set some new configuration option (lets say `PAPERLESS_COOKIE_NAME`) on both instances to different values. I can do that.

However, we have some ideas about how to handle multi-user scenarios. I don't have a use for that, so I don't exactly know how to approach that. If you'd like to share some insight in what would be required by you to run a single instance of paperless for both of you, please do so in the discussion thread (https://github.com/jonaswinkler/paperless-ng/discussions/52)!

**Comment by praul** [comment]
PAPERLESS_COOKIE_NAME option would be great.
As for now, I think running multiple instances is way easier than a multi-user single install. 
With multiple instances I can
- use custom consume folders, so I can send it to the right person at the moment I'm scanning
- use separated paper-archives easily
and:
- if some person somehow moves away, I can just take the docker-volumes to migrate.

Multi-instance is perfect for me, with the cookie problem fixed :)

**Comment by jonaswinkler** [comment]
Alright, I'll have that added soon.
